content/en/docs/reference/kube-scheduler/policies.md

```diff
@@ -0,0 +1,124 @@
+---
+title: Kube-scheduler Policies
+content_template: templates/concept
+weight: 10
+---
+
+{{% capture overview %}}
+
+A scheduling Policy can be used to specify the predicates and priorities that
+[kube-scheduler](/docs/reference/command-line-tools-reference/kube-scheduler/)
+runs to [filter and score nodes](/docs/concept/scheduling/kube-scheduler/#kube-scheduler-implementation),
```

Comment:
@alculquicondor , Would you double check the link on line 11? I think I received a 404.

Reply:
Fixed